tools/app-provisioning-tool/app-provisioning-lib/ProjectDescriptions/dotnet-blazorwasm-hosted.json

```diff
@@ -3,225 +3,6 @@
   "ProjectRelativeFolder": "",
   "BasedOnProjectDescription": "",
   "ConfigurationProperties": [
```

Comment:
No need for the toplevel blazorwasm hosted to get properties. They are delegated to the sub folders